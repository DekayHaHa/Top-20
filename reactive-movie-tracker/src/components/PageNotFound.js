import React from 'react';

export const PageNotFound = () => {
	return (
		<div>
			<h1>Page not Found</h1>
		</div>
	)
}