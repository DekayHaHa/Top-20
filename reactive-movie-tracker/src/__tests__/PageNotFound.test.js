import React from 'react';
import { shallow } from "enzyme";
import PageNotFound from '../containers/PageNotFound'


describe("PageNotFound", () => {
	let wrapper;
	beforeEach(() => {
		wrapper = shallow(<PageNotFound />);
	});


	it("should match the snapshot with all data passed in correctly", () => {
		expect(wrapper).toMatchSnapshot();
	});

});