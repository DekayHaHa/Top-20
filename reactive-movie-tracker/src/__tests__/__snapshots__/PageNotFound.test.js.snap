// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound should match the snapshot with all data passed in correctly 1`] = `
<div>
  <h1>
    Page not Found
  </h1>
</div>
`;
